webpack.playground.config.js

```diff
@@ -11,14 +11,15 @@ const WIDGET_RC = '.widgetrc';
 
 let widgetRc = {
   widgetOptions: {
-    baseUrl: 'https://{yourOktaDomain}',
+    baseUrl: 'http://localhost:3000',
     logo: '/img/logo_widgico.png',
     logoText: 'Windico',
     features: {
       router: true,
```

Comment:
Does these features make sense in idx flows ?

Reply:
probably not. the `widgetrc` is intend for both v1/v2 and we didn't clearly call out the difference.